scripts/resolve_bugs.pl

```diff
@@ -21,6 +22,10 @@
 
 my ($product, $component, $comment);
 my $resolution = 'WONTFIX';
+my $buglist;
+
+uri_protocol_ok( $uri, 'https');
```

Comment:
Oh! I see. When I said test, I just meant like the existing code that does "die" with some sort of specific error message. This would be good if we were writing tests, but not so much for an interactive script